shared/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1162,9 +1162,10 @@ Undocumented.prototype.readSite = function( query, fn ) {
 	this.wpcom.req.get( '/read/sites/' + query.site, params, fn );
 };
 
-Undocumented.prototype.readSiteFeatured = function( siteId, fn ) {
+Undocumented.prototype.readSiteFeatured = function( siteId, query, fn ) {
+	var params = omit( query, [ 'before', 'after' ] );
```

Comment:
I'm going to leave this one alone to maintain consistency with the rest of the module for now.